web_responsive/static/src/css/web_responsive.scss

```
@@ -202,12 +248,19 @@ $chatter_zone_width: 35%;
             height: 100%;
 
             .search-input {
-                height: 3em;
+                height: 100%;
             }
 
             .search-results {
-                height: calc(100% - 3em);
+                max-height: calc(100vh - 47px - 4em);
+                overflow-y: hidden;
+                overflow-x: scroll;
                 overflow: auto;
+                background: url("../img/home-menu-bg-overlay.svg"),
+                    linear-gradient(to bottom, #f9f9f9, #ffffff);
```

Better to use theme variables here:

```suggestion
                    linear-gradient(to bottom, gray('200'), gray('white'));
```